Sources/Logging/Strings/ConfigureStrings.swift

```diff
@@ -68,6 +70,10 @@ extension ConfigureStrings: CustomStringConvertible {
                 "trying to get the default instance. More info here: https://errors.rev.cat/configuring-sdk"
         case .sdk_version(let sdkVersion):
             return "SDK Version - \(sdkVersion)"
+        case .invalidAPIKey:
+            return "The specified API Key is not recognized.\n" +
+            "Ensure that you are using the public app-specific API key.\n" +
+            "See https://docs.revenuecat.com/docs/authentication for more details"
```

Comment:
cc @codykerns

Reply:
Oh last thing, @aboedo could you help me generating a short version of this URL? 🙏🏻

Reply:
I missed this comment! Just a sec

Reply:
https://rev.cat/auth

Reply:
feel free to merge after updating!

Reply:
Yay thanks!